rJavaPackageExample.Rproj

```diff
@@ -0,0 +1,21 @@
+Version: 1.0
```

Comment:
Did this file slipped through ignore file or is it manually created and required?